docs/src/pages/components/buttons/UseButton.js

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Stack from '@material-ui/core/Stack';
+import { useButton } from '@material-ui/unstyled/ButtonUnstyled';
+import { styled } from '@material-ui/system';
+
+const StyledButtonRoot = styled('span')(`
+  background-color: #007fff;
+  padding: 16px 20px;
+  border-radius: 10px;
+  display: inline-flex;
+  align-items: center;
+  color: #fff;
+  font-weight: bold;
+  transition: all 200ms ease;
+  cursor: pointer;
+  box-shadow: 0 0 0 0 rgba(0, 127, 255, 0);
+
+  &:hover {
+    background-color: #0059b2;
+  }
+
+  &.active {
+    background-color: #004386;
+  }
+
+  &.focusVisible {
+    box-shadow: 0 0 0 5px rgba(0, 127, 255, 0.5);
+    outline: none;
+  }
+
+  &.disabled {
+    opacity: 0.5;
+    pointer-events: none;
+  }
+`);
+
+function CustomButton(props) {
+  const { children } = props;
```

Comment:
Interesting handling of children. I wonder if you consider `{...button.getRootProps()}` take care of it, to remove the gymnastic.

Reply:
My view is the less the hook will deal with the layout, the better. And setting `children` is forcing a specific layout. Developers should be free to wrap the children in any custom markup, so generally, I would leave it as it is.